plugin/trino-hive/src/main/java/io/trino/plugin/hive/HiveSessionProperties.java

```diff
@@ -387,6 +394,27 @@ public HiveSessionProperties(
                         "Require filter on partition column",
                         hiveConfig.isQueryPartitionFilterRequired(),
                         false),
+                new PropertyMetadata<>(
+                        QUERY_PARTITION_FILTER_REQUIRED_SCHEMAS,
+                        "List of schemas for which filter on partition column is enforced.",
+                        new ArrayType(VARCHAR),
+                        Set.class,
+                        ImmutableSet.of(),
```

Comment:
add a config

Reply:
If we specify it via a config then as default it would apply filter only on those schemas ? Won't it conflict it there ?